src/Uno.UI/UI/Xaml/Controls/TextBlock/TextBlock.cs

```diff
@@ -934,5 +934,7 @@ internal override void UpdateThemeBindings()
 
 			SetDefaultForeground(ForegroundProperty);
 		}
+
+		internal override bool CanHaveChildren() => true;
```

Comment:
Why is this true for TextBlock?

Reply:
In UWP `TextBlock` does have children in some form, this is taken from the sources. While it is not necessarily this way in our case, it does not cause issues.